Makefile

```diff
@@ -15,6 +15,10 @@
 # limitations under the License.
 #
 
+# DO NOT RUN
+# This makefile is for ci test and have dependency, do not run locally
```

Comment:
# The makefile is for commit test and has dependencies. Do not run locally. 
# If you want to run the unit tests, run command `go test ./...` directly.